sequoia/settings/offline_rl/setting.py

```diff
@@ -0,0 +1,110 @@
+import d3rlpy
+import gym
+from gym.wrappers import RecordEpisodeStatistics
+from matplotlib import pyplot as plt
+from sklearn.model_selection import train_test_split
+from typing import ClassVar, List, Tuple, Dict, Any
+
+from sequoia import Results
+from sequoia.settings.base import Setting
+from torch.utils.data import DataLoader
+from sequoia.settings.base import Method
+from dataclasses import dataclass
+
+from simple_parsing.helpers import choice
+
+
+
+@dataclass
+class OfflineRLResults(Results):
+
+    # TODO: Write these methods
+    def summary(self) -> str:
+        return f"Offline RL results: {self.objective_name} = {self.objective}"
+
+    def make_plots(self) -> Dict[str, plt.Figure]:
+        return {}
+
+    def to_log_dict(self, verbose: bool = False) -> Dict[str, Any]:
+        return {}
```

Comment:
This should return some information, something like
```suggestion
        return {self.objective_name: self.objective}
```
Ideally this would also have a bit more info, but that should do for now.